index.html

```diff
@@ -11,169 +11,223 @@
-        </nav>
-      </footer>
+                  <section class="display-project4">
+                      <div class="placeholder-div">
+                          <p><img src="image/Img-Placeholder.png" alt="image of project" class="ph-image"></p>
+                          
+
+                      </div>
+                      <div class="project-text-nav">
+                          <h2 class="hh2">Multi-Post Story</h2>
+                          <p class="flex-item holdtext">
+                              A daily selection of privately personalized reads; 
+                              no accounts or sign-ups required. 
+                              has been the industry's standard dummy text ever since the 1500s, 
+                              when an unknown printer took a standard dummy text.
+                          </p>
+                          <ul class="flex-item navlang-flex">
+                              <li><a href="#" class="subflex-item artag">CSS</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">html</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">bootstrap</a></li>
+                              <li><img src="image/css-divider.png" alt="divider image" class="subflex-item"></li>
+                              <li><a href="#" class="subflex-item artag">Ruby</a></li>
+                              </ul>
+                          <ul class="seep"><li><a href="#" class="subflex-item seeproject">See Project</a></li></ul>
+                      </div>
+                      
+                  </section>
 
+              </div>
+  
+          </article>
+          <article class="contact-section">
+              <section class="cc-sect cc-top">
+                  <p class="contact mble">CONTACT US</p>
+                  <p class="contact dstop">Let's chat!</p>
+                  <p class="text-invite">
+                      If you have an application you are interested in developing, 
+                      a feature that you need built or a project that needs coding. 
+                       love to help with it 
+                  </p>
+              </section>
+              <section class="cc-sect">
+                  <form action="https://formspree.io/f/meqdogdz" method="post" class="myform" id="someform">
+                      <input type="text" maxlength="30" name="Name" required placeholder="Full Name">
+                      <input type="email" name="Email" required placeholder="Email" id="email">
+                      <textarea maxlength="500" name="message" required placeholder="Enter You Message Here"></textarea>
+                      <button type="submit">See Project</button>
```

Comment:
Kindly changes the text of this button to say `Send` instead of `See Project` as shown in Figma design 🤗
Also kindly fix the color of text to be white as shown in Figma design.

Your design:

![image](https://user-images.githubusercontent.com/73714615/187038668-9635ad8d-5dce-4517-b7fb-7aef5d04322c.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187038660-b1b47014-bdfd-41f3-b0ea-4ade9fb204cf.png)